src/main/java/com/bio4j/angulillos/titan/TitanUntypedGraph.java

```diff
@@ -7,20 +7,21 @@
 
 import com.bio4j.angulillos.*;
 import static com.bio4j.angulillos.conversions.*;
+
 import com.thinkaurelius.titan.core.*;
 import com.thinkaurelius.titan.core.schema.*;
-import com.tinkerpop.blueprints.Edge;
+
+import org.apache.tinkerpop.gremlin.structure.Direction;
+
 
 public interface TitanUntypedGraph extends UntypedGraph<TitanVertex,VertexLabelMaker,TitanEdge,EdgeLabelMaker> {
 
 
   // TitanGraphQuery<? extends TitanGraphQuery> query();
   TitanGraph titanGraph();
-  default TitanManagement managementSystem() { return titanGraph().getManagementSystem(); }
-
-  default void commit() { titanGraph().commit(); }
+  default TitanManagement managementSystem() { return titanGraph().openManagement(); }
```

Comment:
`.commit()` is removed from here, because it has to be called on the management system:
```java
TitanManagement mgmt = graph.managementSystem();
...
mgmt.commit();
```